### PR TITLE
Clearing up traits for S9 and adding it is neither P194 nor P205

### PR DESCRIPTION
**Pull request by david20000813**
The positive changes in this PR are:

1. Adding S9 is not P194;
2. Adding S9 is not P205;
3. Fixing the proof that S9 is not P33 - the original proof contains some minor typos;
4. Fixing the proof that S9 is not P43. The original proof doesn't make sense as it claims a singleton set is not arc connected. The referenced item in _Counterexamples in Topology_ also has nothing to do with P43.

All the other changes in this PR are simply removal of redundant traits.

### CODE REVIEW

File: spaces/S000009/properties/P000033.md
Comment (david20000813) [inline spaces/S000009/properties/P000033.md:10]:
```suggestion
Pick a countably infinite subset $C \subset X \setminus \{p\}$. The collection of all sets of the form $\{x,p\}$ for $x \in C$, along with $X \setminus C$, is a countable open cover with no point-finite open refinement that covers $X$.
```

**Review by GeoffreySangston — APPROVED** [review]
The proofs look right to me. Maybe wait for a second review though. Edit: I think it's fine actually.

**Pull request merged**
